bin/stashcp2/stashcp.py

```diff
@@ -0,0 +1,219 @@
+import optparse
+import sys
+import subprocess
+import datetime
+import time
+import re
+import os
+import json
+import multiprocessing
+import urllib2
+
+parser = optparse.OptionParser()
+parser.add_option('--debug', dest='debug', action='store_true', help='debug')
+parser.add_option('-r', dest='recursive', action='store_true', help='recursively copy')
+parser.add_option('--closest', action='store_true')
+args,opts=parser.parse_args()
+
+def find_closest():
+    closest=subprocess.Popen(['./get_best_stashcache.py', '0'], stdout=subprocess.PIPE)
```

Comment:
Same, why are you calling another python script?  Create a module, class, or function to do this!